src/chapters/data-types/studio.rst

```diff
@@ -0,0 +1,47 @@
+.. _area-of-a-circle-studio:
+
+Studio: Area of a Circle
+========================
+
+To get started, create a new console application in Visual Studio for the studio.
+
+Calculate the Area of a Circle
+------------------------------
+
+Write a program that prompts the user for the radius of a circle.
+Calculate the area of the user's circle and print the result.
+
+.. admonition:: Tip
+
+   Recall that the area of a circle is ``A = pi * r * r`` where ``pi`` is
+   3.14 and ``r`` is the radius.
+
+Here’s an example of how your program should work:
+
+::
+
+   Enter a radius: 2.5
+   The area of a circle of radius 2.5 is: 19.625
+
+Some questions to ask yourself:
+
+#. What data type should the radius be?
+#. What is the best way to get user input into a variable ``radius`` of
+   that type?
+
+.. admonition:: Note
+
+   Use the ``Math`` class in C# to get the value of pi and square the radius. 
```

Comment:
maybe we could refer to this as System.Math since we talked about the System namespace before and we haven't asked them to pull in another class from it yet, other than Console.

Reply:
Done! Thanks!